.github/scripts/setup-workflow-monitor.py

```diff
@@ -29,27 +23,39 @@ def setup_workflow_monitor(platform: Platform, max_runtime: int) -> None:
         # This generates a file that can be sourced to get all the right keys / ids to run any of the
         # Azure jobs. On testing, the environment variables did not correctly pass themselves to the
         # screen job on
+        assert not RUN_LOCAL, "Workflow monitor setup only works running under GH-A"
 
-        if platform == Platform.AZURE:
-            generate_azure_credited_env()
-            azure_source_string = 'source azure_env.sh;'
-        else:
-            azure_source_string = ''
+        workflow_log = f"{manager_ci_dir}/workflow-monitor-screen.log"
 
-        # Put a baseline time-to-live bound on the manager.
-        # Instances will be terminated (since they are spot requests) and cleaned up in a nightly job.
+        run("echo 'zombie kr' >> ~/.screenrc") # for testing purposes, keep the screen on even after it dies
```

Comment:
```suggestion
        # run("echo 'zombie kr' >> ~/.screenrc") # for testing purposes, keep the screen on even after it dies
```

Reply:
your check below will fail because with `zombie kr` enabled the screen session persists even on failure.

Reply:
Ok, I will remove it. I was also checking if the screen invocation itself was incorrect (in addition to if the underlying session was still open).

Reply:
Done